.gitattributes

```diff
@@ -1,4 +1,4 @@
-* text=auto
+* text=auto eol=lf
```

Comment:
Is this correct? or should it just be `text eol=lf`? I think without the `=auto` it won't ignore binary files?

Reply:
since git 2.10, ye, so this is correct
https://github.com/git/git/blob/master/Documentation/RelNotes/2.10.0.txt#L248